[
  {
    "number": 33,
    "title": "The image stops moving after 48 seconds",
    "created_at": "2024-07-12T18:22:52Z",
    "closed_at": "2024-07-13T07:19:16Z",
    "labels": [],
    "url": "https://github.com/antgroup/echomimic/issues/33",
    "body": "Is the maximum length only 48 seconds? If it exceeds 48 seconds, the image will stop moving.",
    "comments_url": "https://api.github.com/repos/antgroup/echomimic/issues/33/comments",
    "author": "TryingToDoBetter25",
    "comments": [
      {
        "user": "oisilener1982",
        "created_at": "2024-07-13T03:15:00Z",
        "body": "Not sure if this will solve your problem but you can try to use gradio and adjust the length"
      },
      {
        "user": "TryingToDoBetter25",
        "created_at": "2024-07-13T07:19:11Z",
        "body": "> Not sure if this will solve your problem but you can try to use gradio and adjust the length\r\n\r\nIt's working,thanks a lot"
      }
    ]
  },
  {
    "number": 20,
    "title": "\u8fd0\u884c    python -u infer_audio2vid.py \u51fa\u73b0\u9519\u8bef ImportError: cannot import name 'PositionNet' from 'diffusers.models.embeddings' (",
    "created_at": "2024-07-11T10:41:31Z",
    "closed_at": "2024-07-14T04:53:30Z",
    "labels": [],
    "url": "https://github.com/antgroup/echomimic/issues/20",
    "body": "Traceback (most recent call last):\r\n  File \"E:\\ai3\\EchoMimic\\infer_audio2vid.py\", line 23, in <module>\r\n    from src.models.unet_2d_condition import UNet2DConditionModel\r\n  File \"E:\\ai3\\EchoMimic\\src\\models\\unet_2d_condition.py\", line 18, in <module>\r\n    from diffusers.models.embeddings import (\r\nImportError: cannot import name 'PositionNet' from 'diffusers.models.embeddings' (E:\\ai3\\EchoMimic\\venv\\Lib\\site-packages\\diffusers\\models\\embeddings.py)\r\n",
    "comments_url": "https://api.github.com/repos/antgroup/echomimic/issues/20/comments",
    "author": "anstonjie",
    "comments": [
      {
        "user": "yuange250",
        "created_at": "2024-07-11T10:42:17Z",
        "body": "pip install diffusers==0.24.0"
      }
    ]
  },
  {
    "number": 19,
    "title": "Is there anyone who set up the enviroment successfully?",
    "created_at": "2024-07-11T10:05:58Z",
    "closed_at": "2024-07-12T07:20:11Z",
    "labels": [],
    "url": "https://github.com/antgroup/echomimic/issues/19",
    "body": "```\r\nconda create -n echomimic python=3.8\r\nconda activate echomimic\r\npip install -r requirements.txt\r\n```\r\nThe above code does not works well for me, and the following errors is given to me,  anyone can help me, or give a docker img to others. \r\n```\r\n\r\nraceback (most recent call last):\r\n  File \"/ssdcache/jtmeng/miniconda3/envs/echomimic/lib/python3.8/site-packages/diffusers/utils/import_utils.py\", line 808, in _get_module\r\n    return importlib.import_module(\".\" + module_name, self.__name__)\r\n  File \"/ssdcache/jtmeng/miniconda3/envs/echomimic/lib/python3.8/importlib/__init__.py\", line 127, in import_module\r\n    return _bootstrap._gcd_import(name[level:], package, level)\r\n  File \"<frozen importlib._bootstrap>\", line 1014, in _gcd_import\r\n  File \"<frozen importlib._bootstrap>\", line 991, in _find_and_load\r\n  File \"<frozen importlib._bootstrap>\", line 961, in _find_and_load_unlocked\r\n  File \"<frozen importlib._bootstrap>\", line 219, in _call_with_frames_removed\r\n  File \"<frozen importlib._bootstrap>\", line 1014, in _gcd_import\r\n  File \"<frozen importlib._bootstrap>\", line 991, in _find_and_load\r\n  File \"<frozen importlib._bootstrap>\", line 975, in _find_and_load_unlocked\r\n  File \"<frozen importlib._bootstrap>\", line 671, in _load_unlocked\r\n  File \"<frozen importlib._bootstrap_external>\", line 843, in exec_module\r\n  File \"<frozen importlib._bootstrap>\", line 219, in _call_with_frames_removed\r\n  File \"/ssdcache/jtmeng/miniconda3/envs/echomimic/lib/python3.8/site-packages/diffusers/models/autoencoders/__init__.py\", line 1, in <module>\r\n    from .autoencoder_asym_kl import AsymmetricAutoencoderKL\r\n  File \"/ssdcache/jtmeng/miniconda3/envs/echomimic/lib/python3.8/site-packages/diffusers/models/autoencoders/autoencoder_asym_kl.py\", line 21, in <module>\r\n    from ..modeling_outputs import AutoencoderKLOutput\r\n  File \"/ssdcache/jtmeng/miniconda3/envs/echomimic/lib/python3.8/site-packages/diffusers/models/modeling_outputs.py\", line 7, in <module>\r\n    class AutoencoderKLOutput(BaseOutput):\r\n  File \"/ssdcache/jtmeng/miniconda3/envs/echomimic/lib/python3.8/site-packages/diffusers/utils/outputs.py\", line 61, in __init_subclass__\r\n    import torch.utils._pytree\r\nModuleNotFoundError: No module named 'torch.utils._pytree'\r\n\r\n```",
    "comments_url": "https://api.github.com/repos/antgroup/echomimic/issues/19/comments",
    "author": "Mengjintao",
    "comments": [
      {
        "user": "yuange250",
        "created_at": "2024-07-11T10:15:52Z",
        "body": "\u8fd9\u79cd\u4e00\u822c\u662fconda\u5df2\u7ecf\u6709\u4e86\u73b0\u6210\u7684torch\u548cdiffusers\u7248\u672c\uff0c\u8ddf\u65b0\u88c5\u7684\u5305\u4e0d\u9002\u914d\uff0c\u53ef\u4ee5\u8bd5\u8bd5\u91cd\u88c5\u4e0b\uff1a\r\npip install torch==2.0.1\r\npip install torchvision==0.15.2\r\npip install diffusers==0.24.0"
      },
      {
        "user": "Mengjintao",
        "created_at": "2024-07-11T11:28:07Z",
        "body": "It works great!  Thanks for your help. "
      }
    ]
  },
  {
    "number": 10,
    "title": "ImportError: cannot import name 'PositionNet' from 'diffusers.models.embeddings' ",
    "created_at": "2024-07-10T16:02:33Z",
    "closed_at": "2024-07-11T09:01:40Z",
    "labels": [],
    "url": "https://github.com/antgroup/echomimic/issues/10",
    "body": "(echomimic) C:\\ALLWEBUI\\EchoMimic\\EchoMimic>python -u infer_audio2vid.py\r\nTraceback (most recent call last):\r\n  File \"infer_audio2vid.py\", line 23, in <module>\r\n    from src.models.unet_2d_condition import UNet2DConditionModel\r\n  File \"C:\\ALLWEBUI\\EchoMimic\\EchoMimic\\src\\models\\unet_2d_condition.py\", line 18, in <module>\r\n    from diffusers.models.embeddings import (\r\nImportError: cannot import name 'PositionNet' from 'diffusers.models.embeddings' (C:\\Users\\admin\\.conda\\envs\\echomimic\\lib\\site-packages\\diffusers\\models\\embeddings.py)",
    "comments_url": "https://api.github.com/repos/antgroup/echomimic/issues/10/comments",
    "author": "huanggou666",
    "comments": [
      {
        "user": "farhanyosuf11",
        "created_at": "2024-07-10T16:51:22Z",
        "body": "I fixed it by installing diffusers=0.24.0"
      }
    ]
  }
]